### ISSUE TITLE
Handle API Errors Gracefully

### ISSUE THREAD

**Issue by teddyjfpender** [palladians/pallad#127]
# Handling Network Errors 🚨 

Sometimes, when querying for a wallet's state from an API, the response might be an error, leaving the wallet unable to obtain the state it expects (e.g. `AccountInfo`). One known instance of this impacting users is in prevent users from progressing/completing the onboarding (either restoring or creating a new wallet).

## Expected behaviour
Gracefully handle API errors that allow users to continue onboarding. Signifiers should communicate to the user issues/errors if they are relevant -- good interaction design is paramount.

## Example Logs
When creating a new wallet and typing in the correct word to prove knowledge of the created mnemonic, users are left stuck at the onboarding screen with `Next` button highlighted but not progressing to the `Overview` page. Console's logs:
![image](https://github.com/palladians/pallad/assets/92999717/6454145d-c9c9-4660-8e79-07385cce6e89)

**Comment by FilipHarald** [comment]
I agree that allowing user to continue onboarding would be optimal. But a first step could be to just display that there is a connection error.
